src/main/java/com/example/ticketbookingapp/service/SeanceServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.example.ticketbookingapp.service;
+
+import com.example.ticketbookingapp.model.Seance;
+import com.example.ticketbookingapp.repository.SeanceRepository;
+import org.springframework.stereotype.Service;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Service
+public class SeanceServiceImpl {
+
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd H:m:s");
```

Comment:
czemu nie format iso?

Reply:
Nie wiedziałem z jakiego formatu skorzystać i zamiast skorzystać z dostępnego formatu niepotrzebnie stworzyłem swój formatter, teraz rozwiązał bym to w ten sposób:
private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ISO_DATE_TIME;